managed/services/management/dbaas/pxc_cluster_service.go

```diff
@@ -35,12 +35,14 @@ import (
 )
 
 const (
-	pxcDefaultClusterSize   = 3
-	pxcDefaultCPUM          = 1000
-	pxcDefaultMemoryBytes   = 2000000000
-	pxcDefaultDiskSize      = 25000000000
-	proxyDefaultCpuM        = 1000
-	proxyDefaultMemoryBytes = 2000000000
+	pxcDefaultClusterSize      = 3
+	pxcDefaultCPUM             = 1000
+	pxcDefaultMemoryBytes      = 2000000000
+	pxcDefaultDiskSize         = 25000000000
+	proxyDefaultCPUM           = 500
+	proxyDefaultMemoryBytes    = 500000000
+	proxySQLDefaultCPUM        = 1000
+	proxySQLDefaultMemoryBytes = 2000000000
```

Comment:
To be honest it's not obvious from the first sight what is the difference between these variables. DBasS always use haproxy? If yes, maybe, we can name variables like `haproxyDefaultCPUM` and `haproxyDefaultMemoryBytes`?

Reply:
After last discussion with Max Bubenick we decided to use same defaults for proxysql and for haproxy. I reverted changes